src/browser/ui/React.js

```diff
@@ -41,7 +42,7 @@ var createElement = ReactElement.createElement;
 var createFactory = ReactElement.createFactory;
 var cloneElement = ReactElement.cloneElement;
 
-if (__DEV__) {
+if (__DEBUG__) {
```

Comment:
this should stay dev-only, as it's about validating proptypes I think?  Or am I misremembering something?